### DISCUSSION TITLE
Can SWISS support the function of connecting the component cable directly to the VGA port?

### DISCUSSION THREAD

**Discussion by Fanturner** [emukidid/swiss-gc#688] [Ideas]
I have NGC Component Cable and gcvideo HDMI Adapter Converter, CRT monitor that supports SOG. NGC uses the Component Cable to connect this monitor directly to display a green screen. I know the VGA signal is missing. I know that the component cable can DIY VGA port, I also know that GCvideo can output VGA signal, but it needs some diy ability.
PS2, OG xbox, these two consoles have achieved 480p VGA signal output through GSM and BIOS writing.
SWISS is so powerful, can it also make NGC output VGA signal easily?
Thanks to the swiss development team.

**Comment by Extrems** [comment]
No.

**Reply by Fanturner** [reply]
> No.
All right，It's a pity....I continue to use the HDMI to VGA converterTAT
THX for reply.
